ENGAGEHF/Dashboard/ToDo/Views/ToDoSection.swift

```diff
@@ -0,0 +1,63 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import FirebaseAuth
+import FirebaseFirestore
+import SpeziViews
+import SwiftUI
+
+
+struct ToDoSection: View {
```

Comment:
I am wondering if we should use the same manager setup as for the Notifications here as we do for the Notifications. It might be that we want to do that in a separate PR or include this in this PR.